pyproject.toml

```diff
@@ -19,6 +19,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.7"
 nautobot = "^1.2.0"
+setuptools = "^65.6.3"
```

Comment:
Is this a temporary workaround for an issue in newer setuptools versions? Will we want to remove it soon? Does it conflict with any version of Nautobot's dependencies?